packaging/prerelease/s3_index.sh

```diff
@@ -41,9 +41,3 @@ fi
 
 echo "Checking if we need to promote a release for testing ($platform)"
 "$release_bin" promote-test-releases --bucket-name="$bucket_name" --platform="$platform"
```

Comment:
Nope... leave this here... I will probably nuke this as some point and have every build auto promote to testing.